openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/LightGBMResultTest.java

```diff
@@ -0,0 +1,286 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.feedzai.openml.provider.lightgbm;
+
+import com.feedzai.openml.data.Dataset;
+import com.feedzai.openml.data.Instance;
+import com.feedzai.openml.data.schema.DatasetSchema;
+import com.feedzai.openml.data.schema.FieldSchema;
+import com.feedzai.openml.data.schema.NumericValueSchema;
+import com.feedzai.openml.provider.exception.ModelLoadingException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.file.Files.createTempFile;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+/**
+ * Test class to ensure that Microsoft LightGBM predicts the expected
+ * <i>class distribution</i> and <i>feature contributions</i>.
+ *
+ * @author Sheng Wang (sheng.wang@feedzai.com)
+ * @since 1.2.2
+ */
+public class LightGBMResultTest {
+
+    /**
+     * Logger for this class.
+     */
+    private static final Logger logger = LoggerFactory.getLogger(LightGBMResultTest.class);
+
+    /**
+     * Parameters for model train.
+     */
+    private static final Map<String, String> MODEL_PARAMS = getTestParams();
+
+
+    /**
+     * Result Schema:
+     * <ul>
+     *     <li>card -> number</li>
+     *     <li>amount -> number</li>
+     *     <li>cat1_generator -> number</li>
+     *     <li>cat2_generator -> number</li>
+     *     <li>cat3_generator -> number</li>
+     *     <li>num1_float -> number</li>
+     *     <li>num2_float -> number</li>
+     *     <li>num3_float -> number</li>
+     *     <li>fraud_label -> number</li>
+     * </ul>
+     */
+    public static final DatasetSchema SCHEMA = getTestSchema();
+
+    /**
+     * Result Schema:
+     * <ul>
+     *     <li>card -> number</li>
+     *     <li>amount -> number</li>
+     *     <li>cat1_generator -> number</li>
+     *     <li>cat2_generator -> number</li>
+     *     <li>cat3_generator -> number</li>
+     *     <li>num1_float -> number</li>
+     *     <li>num2_float -> number</li>
+     *     <li>num3_float -> number</li>
+     *     <li>bias -> number</li>
+     *     <li>predictions -> number</li>
+     * </ul>
+     */
+    public static final DatasetSchema RESULT_SCHEMA = getTestResultSchema();
+
+    /**
+     * The max number of instance to train, i.e., the number of lines in the file {@link treeshap_t treeshap_train.csv}.
```

Comment:
```suggestion
     * The max number of instances to train, i.e., the number of lines in the file {@link treeshap_t treeshap_train.csv}.
```

Reply:
@shenggwang just to make sure, I noticed that the suggestion was not applied, yet this conversation was marked as resolved. Was that deliberate or an overlook?